cmd/timestamp-server/app/root.go

```diff
@@ -60,8 +62,10 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&logType, "log-type", "dev", "logger type to use (dev/prod)")
 	rootCmd.PersistentFlags().BoolVar(&enablePprof, "enable-pprof", false, "enable pprof for profiling on port 6060")
 
-	rootCmd.PersistentFlags().String("hostname", "timestamp.sigstore.dev", "public hostname of instance")
-	rootCmd.PersistentFlags().String("address", "127.0.0.1", "Address to bind to")
+	// pprof/metrics read and write timeout flags
+	rootCmd.PersistentFlags().DurationVar(&readTimeout, "read-timeout", 10*time.Second, "pprof/metrics read timeout")
```

Comment:
there's an existing read-timeout/write-timeout flag provided - https://github.com/sigstore/timestamp-authority/blob/main/pkg/generated/restapi/server.go#L100-L101

Can we reuse these?

Reply:
Yes, good catch.